cgi/generate_sample_import_file.pl

```diff
@@ -56,16 +56,26 @@
 my $worksheet = $workbook->add_worksheet();
 my %formats = (
 	normal => $workbook->add_format(border => 1, bold => 1),
-	mandatory => $workbook->add_format(border => 1, bold => 1, bg_color => '#aaffcc'),
-	recommended => $workbook->add_format(border => 1, bold => 1, bg_color => '#ccffdd'),
-	optional => $workbook->add_format(border => 1, bold => 1, bg_color => '#eeffee'),
+	mandatory => $workbook->add_format(border => 1, bold => 1, bg_color => '#0f55cc', color => 'white', valign => 'vcenter', align=>'center'),
+	recommended => $workbook->add_format(border => 1, bold => 1, bg_color => '#ccffdd', valign => 'vcenter', align=>'center'),
+	optional => $workbook->add_format(border => 1, bold => 1, bg_color => '#eeffee',  valign => 'vcenter', align=>'center'),
+	description => $workbook->add_format(italic => 1, text_wrap => 1, align => 'left', valign => 'vcenter'),
 );
 
 # Re-use the structure used to output select2 options in import_file_select_format.pl
 my $select2_options_ref = generate_import_export_columns_groups_for_select2([$lc]);
 
 my $headers_row = 0;
-my $col = 0;
+my $description_row = 1;
+my $col = 1;
+
+my $field_id;
+my $comment;
+
+$worksheet->set_row(0, 70);
```

Comment:
It would be good to set a height for the description row as well, as on LibreOffice Calc it's not getting auto-expanded as in your screenshot. Here is what it looks like on my side:

![image](https://github.com/openfoodfacts/openfoodfacts-server/assets/8158668/dd9b2e21-fc04-4052-868b-0a07e823241a)

Reply:
Just changed the colors, can you please check?

Reply:
@anaritadauane the colors look great, thanks!